lib/model/rwfolder.go

```diff
@@ -1495,6 +1491,8 @@ func (f *sendReceiveFolder) finisherRoutine(ignores *ignore.Matcher, in <-chan *
 				blockStats["copyOriginShifted"] += state.copyOriginShifted
 				blockStats["copyElsewhere"] += state.copyTotal - state.copyOrigin
 				blockStatsMut.Unlock()
+			} else {
+				f.newError("finisher", state.file.Name, err)
```

Comment:
This is not even equivalent to what it was, as the error returned by performFinish no longer causes f.newError as it's within the non-error if case now.

Reply:
yes, I just recognized and reverted